<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotFrame</class>
 <widget class="QFrame" name="PlotFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolButtonsHLayout">
     <item>
      <widget class="QToolButton" name="showCurvesToolButton">
       <property name="toolTip">
        <string>Affiche ou masque les tracés de toutes les courbes</string>
       </property>
       <property name="text">
        <string>Afficher les tracés</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/stats_lines.png</normaloff>:/Resources/Icons/stats_lines.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+T</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showPointsToolButton">
       <property name="toolTip">
        <string>Affiche ou masque les points de toutes les courbes</string>
       </property>
       <property name="text">
        <string>Afficher les points </string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/stats_cross.png</normaloff>:/Resources/Icons/stats_cross.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+P</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showLineToolButton">
       <property name="toolTip">
        <string>Afficher une ligne verticale de délimitation</string>
       </property>
       <property name="text">
        <string>Afficher la ligne verticale</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/Line.png</normaloff>:/Resources/Icons/Line.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+L</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomOutToolButton">
       <property name="toolTip">
        <string>Effectuer un zoom arrière</string>
       </property>
       <property name="text">
        <string>Dézoomer</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/zoom_out.png</normaloff>:/Resources/Icons/zoom_out.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="eraseSelectionToolButton">
       <property name="toolTip">
        <string>Effacer les éléments mis en évidence</string>
       </property>
       <property name="text">
        <string>Effacer la sélection</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/eraser.png</normaloff>:/Resources/Icons/eraser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="eraseToolButton">
       <property name="toolTip">
        <string>Effacer toutes les données dans les différentes vues</string>
       </property>
       <property name="text">
        <string>Effacer</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/eraser.png</normaloff>:/Resources/Icons/eraser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+Del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="restoreDefaultSizeToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/resize.png</normaloff>:/Resources/Icons/resize.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="printToolButton">
       <property name="toolTip">
        <string>Imprimer la zone graphique courante</string>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/print.png</normaloff>:/Resources/Icons/print.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ScrollHandDragButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources.qrc">
         <normaloff>:/Resources/Icons/cursor_hand.png</normaloff>:/Resources/Icons/cursor_hand.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="plotLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
